src/substrate/subscribeFunc.ts

```diff
@@ -101,51 +100,46 @@ export const subscribeEvents: SubscribeFunc<ApiPromise, Block, ISubscribeOptions
       offlineRange = { startBlock: lastBlockNumber };
     }
 
+    
     // if we can't figure out when the last block we saw was,
-    // and we are not running in archival mode, do nothing
+    // do nothing
```

Comment:
Why was this comment changed?

Reply:
Archival mode is now independent of `ProcessMissedBlocksFn`. When running in archival mode, we don't execute  `PollMissedBlocksFn`. Since the functionality following this comment was changed, hence updated the comment. Thanks.